canonical-web-and-design/ubuntu.com
# Clean up Prettier and formatting checks

## Done

- Bump Prettier to latest
- Remove all trace of sass-lint as we no longer use it
- Run Prettier over all scss and js files
- Add npm script to check prettier
- Add Github workflow to call npm script on pull request

## QA

- [x] Sanity check code
- [x] Ensure all tests on this PR pass
- [x] Check the test `check-prettier` passes
